tests/test-sources/plugins/by-name/pckr/default.nix

```diff
@@ -8,7 +8,7 @@
       enable = true;
 
       settings = {
-        pack_dir.__raw = "require('pckr.util').join_paths(vim.fn.stdpath('data'), 'site')";
+        pack_dir.__raw = "vim.fs.joinpath(vim.fn.stdpath('data'), 'site')";
```

Comment:
This test is claiming to be "defaults". Is it still?

Do we have declared options for these that also need their `pluginDefault`s updating?

Reply:
Before, they had their internal `util.join_path` function. In https://github.com/lewis6991/pckr.nvim/commit/995d6971ebfcdb93fc3254ffb5c4d998f24c38f0, they switched to the vim-native `vim.fs.joinpath`.